apps/nestjs-app/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "private": true,
   "dependencies": {
+    "@company/node-package": "workspace:*",
```

Comment:
I think this is the intended change a developer makes to bring in a monorepo package? 

All other changes to tsconfig files are done by moon?